Composer/packages/client/src/components/ProjectTree/treeItem.tsx

```diff
@@ -3,13 +3,22 @@
 
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
-import { OverflowSet } from 'office-ui-fabric-react/lib/OverflowSet';
+import { OverflowSet, IOverflowSetItemProps } from 'office-ui-fabric-react/lib/OverflowSet';
 import { IconButton } from 'office-ui-fabric-react/lib/Button';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
 
-import { moreButton, overflowSet, moreMenu, navItem, itemText } from './styles';
+import { moreButton, overflowSet, test, navItem, itemText } from './styles';
```

Comment:
Why is this called test?

Reply:
I forgot to replace it after testing the style